docs/source/algorithms/MatchSpectra-v1.rst

```diff
@@ -0,0 +1,58 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Calculate the factor(s) that best match the individual spectra to a
+reference spectrum using `Gauss-Markov process
+<https://en.wikipedia.org/wiki/Gauss%E2%80%93Markov_process>`_. This
+is the same algorithm used by the "blend" step of `PDFgetN
+<http://pdfgetn.sourceforge.net/>`_.
+
```

Comment:
Should have a formula explaining what you are doing, especially vertical vs horizontal offset.
Also add information about expected failures (such as when all errors are 0)

Reply:
done